kong/plugins/frontier/access.lua

```diff
@@ -123,6 +123,10 @@ function _M.run(conf)
     local user_token = check_request_identity(conf, cookies, bearer)
     kong.service.request.set_header(conf.header_name, user_token)
 
+    if conf.override_authz_header then
+        kong.service.request.set_header("Authorization", "Bearer " .. user_token)
```

Comment:
@kushsharma Continuing from the Slack thread, I feel like this change will be an idempotent one, and we could remove the configuration flag altogether and always set this.